frontend/src/metabase/components/SingleSelectListField/SingleSelectListField.tsx

```diff
@@ -38,7 +38,7 @@ const SingleSelectListField = ({
     createOptionsFromValuesWithoutOptions(value, options),
   );
 
-  const augmentedOptions = useMemo(() => {
+  const augmentedOptions = useMemo<Option[]>(() => {
```

Comment:
Small type improvement since I happen to come across this file, when reproducing the repro. Without this explicit type `augmentedOptions` will be `any[]`